catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/RTL_BricksTest.java

```diff
@@ -0,0 +1,163 @@
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.common.FileChecksumContainer;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.Script;
+import org.catrobat.catroid.content.SingleSprite;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.content.StartScript;
+import org.catrobat.catroid.content.bricks.Brick;
+import org.catrobat.catroid.content.bricks.ComeToFrontBrick;
+import org.catrobat.catroid.content.bricks.GoNStepsBackBrick;
+import org.catrobat.catroid.content.bricks.HideBrick;
+import org.catrobat.catroid.content.bricks.PlaceAtBrick;
+import org.catrobat.catroid.content.bricks.SetSizeToBrick;
+import org.catrobat.catroid.content.bricks.ShowBrick;
+import org.catrobat.catroid.io.StorageHandler;
+import org.catrobat.catroid.test.utils.Reflection;
+import org.catrobat.catroid.ui.MainMenuActivity;
+
+import org.catrobat.catroid.uiespresso.util.assertions.LayoutDirectionAssertions;
+import org.catrobat.catroid.uiespresso.util.assertions.TextDirectionAssertions;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.hamcrest.CoreMatchers.anything;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import static org.hamcrest.core.Is.is;
+
+@RunWith(AndroidJUnit4.class)
+public class RTL_BricksTest {
+	private static final String DEFAULT_TEST_PROJECT_NAME = "testProject";
+	private static final Locale ARABICLOCALE = new Locale("ar");
+
+	@Rule
+	public ActivityTestRule<MainMenuActivity> mActivityRule = new ActivityTestRule<>(MainMenuActivity.class);
+
+	@Before
+	public void setUp() throws Exception {
+		createTestProject(DEFAULT_TEST_PROJECT_NAME);
+	}
+
+	@Test
+	public void assertBricksLayoutAndTextDirectionAreRTL() {
+		assertThat("LayoutDirection is not RTL", true, is(isRTL()));
```

Comment:
please use assertTrue, etc. to increase readability, or at least use assertThat parameters in the right order:
from javadoc:
"""assertThat(java.lang.String reason, T actual, org.hamcrest.Matcher<T> matcher)
          Asserts that actual satisfies the condition specified by matcher."""

so that would make above line look like
assrtThat( isRTL() ,  is(true) )

but id prefer
assertTrue( isRTL() )